ports/nrf/boards/feather52840/pins.csv

```diff
@@ -0,0 +1,48 @@
+P0_00,PA0
```

Comment:
This should be P0_00 after the comma as well, I see this is copied from the pca10056 directory, I screwed it up there, need to fix. Also I noticed that pca10040 is missing P0_00 totally, need to fix that too, damn.

Reply:
Thanks, I just copied directly from your PR and merge, yes. I'll update this, but maybe wait until you finish looking through in case you spot something else.

Reply:
See https://github.com/adafruit/circuitpython/pull/623/commits/d4356f9416d8bd720fbd22017f8de840f5d3cfb1